src/common/network/NetworkUtils.cpp

```diff
@@ -143,7 +143,7 @@ uint16_t NetworkUtils::getAvailablePort() {
 }
 
 
-bool NetworkUtils::ipv4ToInt(const std::string& ipStr, uint32_t& ip) {
+bool NetworkUtils::ipv4ToInt(const std::string& ipStr, int32_t& ip) {
```

Comment:
Why did you make this changes?  Any difference about int and uint?

Reply:
`IPv4` defined in `ThriftTypes.h` is int32_t, they are should be same.